doc/Getting-started/index.md

```diff
@@ -65,543 +59,258 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
+one step further: install Kaa into your environment and create your own Kaa 
+applications! We will give you 
+[something to start with]({{root_url}}Programming-guide/Your-first-Kaa-application). 
+Kaa installation is available in the single node mode or in the distributed mode, 
+as described in the [Installation guide]({{root_url}}Administration-guide/System-installation).
 
-Using a set of pre-packaged server-side _log appenders_, the Kaa server is able to store records to a filesystem, a variety of big data platforms (Hadoop, MongoDB, Cassandra, Oracle NoSQL etc.), or submit them directly to a streaming analytics system. It is also possible to [create a custom log appender***]().
-
-The structure of the collected data is flexible and defined by the [log schema***](). Based on the log schema defined for the Kaa application, Kaa generates an object model for the records and the corresponding API calls in the client SDK. Kaa also takes care of data marshalling, managing temporary data storage on the endpoint, and uploading data to the Kaa server.
-
-For programming practice, see [collecting data from endpoints***]().
-
-**Using notifications**
-Kaa uses _notifications_ to distribute structured messages, posted within _notification topics_, from the server to endpoints. A notification structure is defined by a corresponding [notification schema***]().
-
-Endpoint are subscribed to notification topics, which can be either mandatory or optional. Access to notification topics is automatically granted according to the endpoint’s group membership. Notifications can be sent either to every endpoint subscribed to a topic or to an individual endpoint.
-
-Notifications can be assigned expiration timestamps to prevent their delivery after a certain period of time.
-
-For programming practice, see [using notifications***]().
+# Further reading
 
-**Distributing operational data**
-Kaa allows you to perform operational data updates, such as configuration data updates, from the Kaa server to endpoints. This feature can be used for centralized configuration management, content distribution, etc. Since Kaa works with structured data and constraint types, it guarantees data integrity.
+Use the following guides and references to make the most of Kaa.
 
-The Kaa server monitors the database for changes and distributes updates to endpoints in the incremental form, thus ensuring efficient bandwidth use. The endpoint SDK performs data merging and persistence, as well as notifies the client code about the specific changes made to the data. As a result, the client application knows exactly where in the data structure the changes occurred and can be programmed to react accordingly.
+* **[Key platform features]({{root_url}}Programming-guide/Key-platform-features/)** - 
+Use this reference to learn about features and capabilities of Kaa
+ \( [Endpoint profiling###](#), [Events###](#), [Notifications###](#), 
```

Comment:
Pages are actually there, for example [Data collection](https://github.com/kaaproject/kaa/tree/develop/doc/Programming-guide/Key-platform-features/Data-collection) or [Endpoint profiles](https://github.com/kaaproject/kaa/tree/develop/doc/Programming-guide/Key-platform-features/Endpoint-profiles) 

They are placeholders, but still usable.